### ISSUE TITLE
[Feature]: Create a web interface for the chat client

### ISSUE THREAD

**Issue by rvolosatovs** [enarx/codex#62]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

The existing chat client should be extended to provide a web interface to send and receive messages.

This would allow to overcome the inability to use threads to simultaneously receive and send messages.

**Comment by nickvidal** [comment]
I did some research to find an open source web interface for the chat client, and I found this one:

https://github.com/nickvidal/chatweb

Students, please feel free to adapt this code to connect with the chat client.

**Comment by rvolosatovs** [comment]
> I did some research to find an open source web interface for the chat client, and I found this one:
> 
> [nickvidal/chatweb](https://github.com/nickvidal/chatweb)
> 
> Students, please feel free to adapt this code to connect with the chat client.

@nickvidal work is already underway on this using React as far as I know. I am not sure the client you linked is relevant for the task at hand.

We're looking for something simple and embeddable. It looks like the client assumes an existence of a bot and things like password handling. Neither of these are goals of the project, what we need is a basic networked application, where peers can exchange lines of text and perform assertions on identity of peers by using data provided by Enarx runtime.
Additionally, we should be using a single WebSocket here and not a REST webhook

Could you specify the requirements on how the client should look like if you have any? Thanks

**Comment by nickvidal** [comment]
Ah, yes, we would just use the UI from this one, removing all the extra code and connecting via WebSocket.
But if we already have something underway, that's great!

**Comment by rvolosatovs** [comment]
> Ah, yes, we would just use the UI from this one, removing all the extra code and connecting via WebSocket. But if we already have something underway, that's great!

Yes, we'll add UI styling as the last step. For now, there are still too many unknowns and adding a complete UI would only complicate making progress.

**Comment by dpal** [comment]
Richard will provide the status update in several days.
